README.md

```diff
@@ -20,11 +20,15 @@ Clone this repository:
 ````
 git clone https://github.com/bibaldo/MMM-COVID19.git
 ````
+Install Package
+````
+cd MMM-COVID19
+npm install
```

Comment:
no need to perform this because the module doesn't have any dependencies to be installed